pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -32,7 +32,68 @@
           </p>
           <% } %>
 
-          <% if (locals.assessment.require_honor_code) { %>
+          <% if (locals.assessment.group_work) { %>
```

Comment:
This should use the shared partial that you added to `pages/shared/studentGroupAssessmentHelpers`.

Reply:
Resolved in [8652798](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/86527983f782382ca918fb20f3c9b995e3862239)